packages/@okta/vuepress-site/code/angular/okta_angular_auth_js/index.md

```diff
@@ -124,16 +124,16 @@ export class OktaAuthService {
   async handleAuthentication() {
     const tokenContainer = await this.oktaAuth.token.parseFromUrl();
 
-    this.oktaAuth.tokenManager.add('idToken', tokenContainer.tokens.idToken);
-    this.oktaAuth.tokenManager.add('accessToken', tokenContainer.tokens.accessToken);
+    this.oktaAuth.tokenManager.add('idToken', tokenContainer.tokens.idToken!);
+    this.oktaAuth.tokenManager.add('accessToken', tokenContainer.tokens.accessToken!);
```

Comment:
what's `!` for?

Reply:
Changed to `as` cast
Doc about !: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator